securesystemslib/formats.py

```diff
@@ -227,7 +235,7 @@
 # Supported TUF key types.
 KEYTYPE_SCHEMA = SCHEMA.OneOf(
   [SCHEMA.String('rsa'), SCHEMA.String('ed25519'),
-   SCHEMA.String('ecdsa-sha2-nistp256')])
+   SCHEMA.String('ecdsa-sha2-nistp256'),SCHEMA.String('spx')])
```

Comment:
Shouldn't there be a space after the "," to be consistent with the previous line?